### ISSUE TITLE
Supply action on wrong character

### ISSUE THREAD

**Issue by reamonntiu** [halogenandtoast/ArkhamHorror#1273]
Finn Edwards has the Map, but Ursula Downs is the one that can use the action on Mouth of K'nyan, not Finn.

file: https://arkham-horror-bugs.s3.amazonaws.com/exports/e3a1d1b784133fbfba3deff160ffb01c2f43c5c3cfdab8ca27644e150bf58742.json

**Comment by halogenandtoast** [comment]
Yup, turns out I was asking about the Compass here, not the map, nice catch.
